test/acceptance/auto_auth.py

```diff
@@ -2,12 +2,13 @@
 Auto-auth page (used to automatically log in during testing).
 """
 
+import os
 import re
 import urllib
+
 from bok_choy.page_object import PageObject
 
-import os
-BASE_URL = os.environ.get('BASE_URL')
+ORA_SANDBOX_URL = os.environ.get('ORA_SANDBOX_URL', failobj=os.environ.get('BASE_URL'))
```

Comment:
I was nervous that it wouldn't work but thinking about it some more it seems that it should. I'll take it out and then only add it back in if the tests still fail.